lib/paper_trail/model_config.rb

```diff
@@ -237,19 +236,5 @@ def setup_options(options)
 
       @model_class.paper_trail_options[:meta] ||= {}
     end
-
-    # @api private
-    def valid_keys_for_has_many
-      if ActiveRecord.gem_version >= Gem::Version.new("5.0")
-        ActiveRecord::Associations::Builder::HasMany.valid_options({})
-      else
-        %i[
-          after_add after_remove anonymous_class as autosave before_add
-          before_remove class_name counter_cache dependent extend foreign_key
-          foreign_type inverse_of join_table primary_key source source_type
-          table_name through validate
-        ]
-      end
-    end
```

Comment:
Good catch. :+1:  Sorry, I sure thought I'd removed that already! Maybe it got added back by mistake when I did a rebase.

Reply:
Oh, I see it now. I'd forgotten to push up that commit.